### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by C4Framework** [C4Framework/C4iOS#332]
I just ran https://github.com/Realm/jazzy on the C4iOS project, and it listed documentation at only 59%. Here's the `undocumented.txt` it generated.

```
C4Line.swift
	public class C4Line: C4Polygon
	public override var points : [C4Point]
C4RegularPolygon.swift
	public class C4RegularPolygon: C4Shape
	public override init()
	required public init(coder aDecoder: NSCoder)
C4Size.swift
	public struct C4Size : Equatable, Comparable, CustomStringConvertible
	public var width: Double
	public var height: Double
	public struct C4Size : Equatable, Comparable, CustomStringConvertible
	public init(_ size: CGSize)
	public init(_ size: C4Size)
C4Movie.swift
	public class C4Movie: C4View
	public func handleReachedEnd()
C4Gradient.swift
	public class C4Gradient : C4View
	public var gradientLayer: C4GradientLayer
	public var colors : [C4Color]
	public var locations : [Double]
	public var startPoint : C4Point
	public var endPoint : C4Point
	public convenience init(frame: C4Rect, colors: [C4Color] = [C4Blue, C4Purple], locations: [Double] = [0,1])
C4StoredAnimation.swift
	public class C4StoredAnimation : C4Animation
	public var values = [String: AnyObject]()
C4Filter.swift
	public protocol C4Filter
	var filterName : String { get }
	func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4Bloom.swift
	public struct C4Bloom : C4Filter
	public let filterName = "CIBloom"
	public var radius: Double = 10.0
	public var intensity: Double = 1.0
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4Circle.swift
	public class C4Circle: C4Ellipse
C4Polygon.swift
	public class C4Polygon: C4Shape
	required public init(coder aDecoder: NSCoder)
	public func close()
C4Sharpen.swift
	public struct C4Sharpen : C4Filter
	public let filterName = "CISharpenLuminance"
	public var sharpness: Double = 0.4
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4Font.swift
	public class C4Font : NSObject
C4HueAdjust.swift
	public struct C4Hue : C4Filter
	public let filterName = "CIHueAdjust"
	public var angle: Double = 1.0
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4DotScreen.swift
	public struct C4DotScreen : C4Filter
	public let filterName = "CIDotScreen"
	public var center: C4Point = C4Point()
	public var width: Double = 2.0
	public var angle: Double = 0
	public var sharpness: Double = 0.5
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
UIGestureRecognizer+Closure.swift
	public var translation: C4Vector
	public var velocity: C4Vector
C4View+Border.swift
	public init()
C4View.swift
	public class C4View : NSObject
	public class C4View : NSObject
	public class C4View : NSObject
	public class C4View : NSObject
	public class C4View : NSObject
	public var view : UIView = UIView()
	public override init()
	public dynamic var zPosition : Double
	public var mask: C4View?
C4LinearGradient.swift
	public struct C4LinearGradient : C4Generator
	public let filterName = "CISmoothLinearGradient"
	public var colors: [C4Color] = [C4Color(red: 0.0, green: 0.0, blue: 0.0, alpha: 1.0),C4Color(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)]
	public var points: [C4Point] = [C4Point(),C4Point(1,1)]
	public init() {}
	public func createCoreImageFilter() -> CIFilter
C4Point.swift
	public struct C4Point : Equatable, CustomStringConvertible
	public var x: Double = 0
	public var y: Double = 0
	public struct C4Point : Equatable, CustomStringConvertible
	public init(_ point: CGPoint)
	public init(_ point: C4Point)
C4TextShape.swift
	public class C4TextShape : C4Shape
	public var text: String = ""
	public var font: C4Font
	public override init()
C4ViewAnimation.swift
	public class C4ViewAnimation : C4Animation
	public var delay: NSTimeInterval = 0
	public var animations: () -> Void
	public init(_ animations: () -> Void)
	public func animate()
	public func animate()
C4ShapeLayer.swift
	public static var disableActions = true
	public override func actionForKey(key: String) -> CAAction?
	public func configureOptions()
C4Animation.swift
	public class C4Animation
	public enum Curve
	case Linear
	case EaseOut
	case EaseIn
	case EaseInOut
	public var autoreverses = false
	public var repeatCount = 0.0
	public var repeats: Bool
	public init()
	public func addCompletionObserver(action: () -> Void) -> AnyObject
	public func removeCompletionObserver(observer: AnyObject)
	public func postCompletedEvent()
	public func addCancelObserver(action: () -> Void) -> AnyObject
	public func removeCancelObserver(observer: AnyObject)
	public func postCancelledEvent()
C4Shape.swift
	public class C4Shape : C4View
	public class C4Shape: C4View
	public var shapeLayer: C4ShapeLayer
	public override init()
C4Transform.swift
	public struct C4Transform : Equatable
	public init()
C4MediaObject.swift
	public protocol C4MediaObject: C4Animatable, C4EventSource, NSObjectProtocol
	public protocol C4Animatable {}
C4GradientLayer.swift
	public class C4GradientLayer: CAGradientLayer
	public static var disableActions = true
	public override func actionForKey(key: String) -> CAAction?
C4Vector.swift
	public struct C4Vector : Equatable, CustomStringConvertible
	public var x: Double = 0
	public var y: Double = 0
	public var z: Double = 0
	public func ⋅ (lhs: C4Vector, rhs: C4Vector) -> Double
	public struct C4Vector : Equatable, CustomStringConvertible
	public init(_ point: CGPoint)
C4Image.swift
	public class C4Image : C4View
	public struct Pixel
	public init(gray: Int)
	public init(_ r : Int, _ g: Int, _ b: Int, _ a: UInt8)
	public class C4Image: C4View
	public var originalSize : C4Size
	public class C4Image : C4View
	public class C4Image : C4View
C4Image+Generator.swift
	public func generate(generator: C4Generator)
C4Twirl.swift
	public struct C4Twirl : C4Filter
	public let filterName = "CITwirlDistortion"
	public var center: C4Point = C4Point(150,150)
	public var radius: Double = 300.0
	public var angle: Double = M_PI_4
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4View+KeyValues.swift
	public override func setValue(value: AnyObject?, forKey key: String)
	public override func valueForKey(key: String) -> AnyObject?
C4QuadCurve.swift
	public class C4QuadCurve : C4Curve
	public var controlPoint = C4Point()
C4Star.swift
	public class C4Star: C4Polygon
C4Generator.swift
	public protocol C4Generator
	var filterName : String { get }
	func createCoreImageFilter() -> CIFilter
C4Rectangle.swift
	public class C4Rectangle: C4Shape
	override public func updatePath()
C4Timer.swift
	public final class C4Timer : NSObject
	public internal(set) var step = 0
	public internal(set) var count: Int
	public internal(set) var interval: Double
	public init(interval: Double, count: Int = Int.max, action: () -> ())
	public func fire()
	public func start()
	public func pause()
	public func stop()
C4Rect.swift
	public struct C4Rect : Equatable, CustomStringConvertible
	public var origin: C4Point
	public var size: C4Size
	public var width : Double
	public var height : Double
	public struct C4Rect : Equatable, CustomStringConvertible
	public init(_ rect: CGRect)
	public init(_ rect: C4Rect)
C4Sepia.swift
	public struct C4Sepia : C4Filter
	public let filterName = "CISepiaTone"
	public var intensity: Double = 1.0
	public init() {}
	public init(intensity: Double) { self.intensity = intensity }
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4CanvasController.swift
	public class C4CanvasController : UIViewController
	public override func prefersStatusBarHidden() -> Bool
C4GaussianBlur.swift
	public struct C4GaussianBlur : C4Filter
	public let filterName = "CIGaussianBlur"
	public var radius: Double = 10
	public init() {}
	public init(radius: Double) { self.radius = radius }
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4ColorBurn.swift
	public struct C4ColorBurn : C4Filter
	public let filterName = "CIColorBurnBlendMode"
	public var background: C4Image = C4Image()
	public init() {}
	public func createCoreImageFilter(inputImage: CIImage) -> CIFilter
C4Triangle.swift
	public class C4Triangle: C4Polygon
C4EventSource.swift
	public protocol C4EventSource
	public func on(event notificationName: String, from sender: AnyObject?, run executionBlock: Void -> Void) -> AnyObject
C4Image+Crop.swift
	public func crop(rect: C4Rect)
C4Color.swift
	public let black     = C4Color(red: 0.0, green: 0.0, blue: 0.0, alpha: 1.0)
	public let darkGray  = C4Color(red: 1.0/3.0, green: 1.0/3.0, blue: 1.0/3.0, alpha: 1.0)
	public let lightGray = C4Color(red: 2.0/3.0, green: 2.0/3.0, blue: 2.0/3.0, alpha: 1.0)
	public let white     = C4Color(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
	public let gray      = C4Color(red: 0.5, green: 0.5, blue: 0.5, alpha: 1.0)
	public let red       = C4Color(red: 1.0, green: 0.0, blue: 0.0, alpha: 1.0)
	public let green     = C4Color(red: 0.0, green: 1.0, blue: 0.0, alpha: 1.0)
	public let blue      = C4Color(red: 0.0, green: 0.0, blue: 1.0, alpha: 1.0)
	public let cyan      = C4Color(red: 0.0, green: 1.0, blue: 1.0, alpha: 1.0)
	public let yellow    = C4Color(red: 1.0, green: 1.0, blue: 0.0, alpha: 1.0)
	public let magenta   = C4Color(red: 1.0, green: 0.0, blue: 1.0, alpha: 1.0)
	public let orange    = C4Color(red: 1.0, green: 0.5, blue: 0.0, alpha: 1.0)
	public let purple    = C4Color(red: 0.5, green: 0.0, blue: 0.5, alpha: 1.0)
	public let brown     = C4Color(red: 0.6, green: 0.4, blue: 0.2, alpha: 1.0)
	public let clear     = C4Color(red: 0.0, green: 0.0, blue: 0.0, alpha: 0.0)
	public let C4Pink    = C4Color(red: 1.0, green: 0.0, blue: 0.475, alpha: 1.0)
	public let C4Blue    = C4Color(red: 0.098, green: 0.271, blue: 1.0, alpha: 1.0)
	public let C4Purple  = C4Color(red: 0.0, green: 0.0, blue: 0.541, alpha: 1.0)
	public let C4Grey    = C4Color(red: 0.98, green: 0.98, blue: 0.98, alpha: 1.0)
	public convenience init(_ pattern: String)
	public convenience init?(_ color: C4Color)
	public convenience init(_ color: C4Color)
C4Image+Filter.swift
	public func apply(filter: C4Filter)
	public func apply(filters filters: [C4Filter])
	public func renderFilteredImage()
C4Curve.swift
	public class C4Curve : C4Shape
	public override var center : C4Point
	public override var origin : C4Point
C4AudioPlayer.swift
	public class C4AudioPlayer : NSObject, AVAudioPlayerDelegate
C4Path.swift
	public init()
	public init(path: CGPathRef)
	public var CGPath: CGPathRef
C4Ellipse.swift
	public class C4Ellipse: C4Shape
	override public func updatePath()
C4Checkerboard.swift
	public struct C4Checkerboard : C4Generator
	public let filterName = "CICheckerboardGenerator"
	public var colors: [C4Color] = [C4Color(red: 0.0, green: 0.0, blue: 0.0, alpha: 1.0),C4Color(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)]
	public var center: C4Point = C4Point()
	public var sharpness: Double = 1
	public var width: Double = 5.0
	public init() {}
	public func createCoreImageFilter() -> CIFilter
C4Arc.swift
	public class C4Arc : C4Shape
C4Wedge.swift
	public class C4Wedge : C4Shape

**Comment by C4Framework** [comment]
Just updated this.

**Comment by C4Framework** [comment]
Closed with a ton of PRs ending at #562
